### PR TITLE
First PR : 0.0.2

### PR DESCRIPTION
**Pull request by sybaris**
- Update projects to build with latest version of Blazor
- Migration from blazor preview --> blazor release
- Starting wrapping fadein and fadeout
- New testapp to test with blazor webassembly, and refactoring to make common code between the 2 testaspp

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,6 +75,12 @@ services.AddBlazorQuery();
 
 
 ## Change log
+- **Version 0.0.2 (Sybaris) - 2020-07-29 https://github.com/sybaris/BlazorQuery **
```
Comment (kevinjpetersen) [inline README.md]: I don't think it makes sense to put Contributor names on a Version per Version basis. How about a Contributor Section instead?

File: src/BlazorQuery.Library/BlazorQuery.Library.csproj [hunk 2]
```diff
@@ -1,31 +1,38 @@
...
+    <AssemblyName>BlazorQuery.Library</AssemblyName>
+    <RootNamespace>BlazorQuery.Library</RootNamespace>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
     <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
-    <Version>0.0.1</Version>
-    <Authors>KevinJPetersen</Authors>
-    <RepositoryUrl>https://github.com/kevinjpetersen/BlazorQuery</RepositoryUrl>
-    <PackageProjectUrl>https://github.com/kevinjpetersen/BlazorQuery</PackageProjectUrl>
+    <Version>0.0.2</Version>
+    <Authors>Sybaris (Jean-Pierre Planas), KevinJPetersen</Authors>
```
Comment (kevinjpetersen) [inline src/BlazorQuery.Library/BlazorQuery.Library.csproj]: Same here in terms of Contributor Tags. I think it makes the most sense to just include the Repo Author. We can always add Contributors to the Description?

File: src/BlazorQuery.TestApp.CommonLib/BlazorQuery.TestApp.CommonLib.csproj [hunk 3]
```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk.Razor">
```
Comment (kevinjpetersen) [inline src/BlazorQuery.TestApp.CommonLib/BlazorQuery.TestApp.CommonLib.csproj:1]: What is the idea of renaming the Project to CommonLib?

**Review by kevinjpetersen — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (kevinjpetersen) [reply README.md]: Also no reason to put the URL of the Repo in the Change Log :)

Comment (sybaris) [reply README.md]: Done. I add a section contributors

File: src/BlazorQuery.Library/BlazorQuery.Library.csproj (hunk 2 above)
Comment (sybaris) [reply src/BlazorQuery.Library/BlazorQuery.Library.csproj]: Done. I set again thr url to your repo.

File: src/BlazorQuery.TestApp.CommonLib/BlazorQuery.TestApp.CommonLib.csproj (hunk 3 above)
Comment (sybaris) [reply src/BlazorQuery.TestApp.CommonLib/BlazorQuery.TestApp.CommonLib.csproj:1]: I have split the TestApp projet in 3 projects. I have follow the principles of the course https://www.pluralsight.com/courses/getting-started-blazor : 1 project for blazor webassembly, 1 project for blazor server, and 1 project that contains all the common code. So CommonLib is the code shared by the 2 projects...

**Review by kevinjpetersen — APPROVED** [review]

**Pull request merged**